portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Salvador's Portfolio</h1>
+      <p>This is Salvador's portfolio.</p>
+      <p>I'm currently a STEP intern at Google from The University of Texas at El Paso.</p>
+      <img src = "/Images/Me.jpg" width="450" height="600">
```

Comment:
Don't hard code styles like width and height in HTML. Use a class `<img src... class="myPortrait" />` and then set the style in your css `.myPortrait { width: 450px; height: 600px; }` This is so that you can swap out the css to change the look of your entire site.

Reply:
Got it!! I'll try to solve that!!